src/proposals/20231115/SparkEthereum_20231115.t.sol

```diff
@@ -0,0 +1,230 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import '../../SparkTestBase.sol';
+
+import { Domain, GnosisDomain } from 'xchain-helpers/testing/GnosisDomain.sol';
+
+import { SparkEthereum_20231115 } from './SparkEthereum_20231115.sol';
+import { SparkGnosis_20231115 } from   './SparkGnosis_20231115.sol';
+
+interface IL2BridgeExecutor {
+    function execute(uint256 index) external;
+}
+
+contract SparkEthereum_20231115Test is SparkEthereumTestBase {
+
+    address public constant WBTC                   = 0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599;
+    address public constant WBTC_PRICE_FEED        = 0x1b44F3514812d835EB1BDB0acB33d3fA3351Ee43;
+    address public constant GNOSIS_BRIDGE_EXECUTOR = 0xc4218C1127cB24a0D6c1e7D25dc34e10f2625f5A;
+
+    uint256 public constant OLD_RETH_SUPPLY_CAP            = 60_000;
+    uint256 public constant NEW_RETH_SUPPLY_CAP            = 80_000;
+    uint256 public constant OLD_WSTETH_SUPPLY_CAP          = 400_000;
+    uint256 public constant NEW_WSTETH_SUPPLY_CAP          = 800_000;
+    uint256 public constant OLD_DAI_LTV                    = 1;
+    uint256 public constant NEW_DAI_LTV                    = 0;
+    uint256 public constant WETH_OPTIMAL_USAGE_RATIO       = 0.90e27;
+    uint256 public constant OLD_WETH_BASE_RATE             = 0.01e27;
+    uint256 public constant NEW_WETH_BASE_RATE             = 0;
+    uint256 public constant OLD_WETH_VARIABLE_RATE_SLOPE_1 = 0.028e27;
+    uint256 public constant NEW_WETH_VARIABLE_RATE_SLOPE_1 = 0.032e27;
+    uint256 public constant OLD_WETH_VARIABLE_RATE_SLOPE_2 = 1.200e27;
+    uint256 public constant NEW_WETH_VARIABLE_RATE_SLOPE_2 = 1.232e27;
+    uint256 public constant NEW_WBTC_SUPPLY_CAP            = 3_000;
+    uint256 public constant NEW_WBTC_BORROW_CAP            = 2_000;
+    uint256 public constant NEW_WBTC_LTV                   = 70_00;
+    uint256 public constant NEW_WBTC_LIQ_THRESHOLD         = 75_00;
+    uint256 public constant NEW_WBTC_LIQ_BONUS             = 7_00;
+    uint256 public constant NEW_WBTC_OPTIMAL_USAGE_RATIO   = 60_00;
+    uint256 public constant NEW_WBTC_BASE_RATE             = 0;
+    uint256 public constant NEW_WBTC_VARIABLE_RATE_SLOPE_1 = 2_00;
+    uint256 public constant NEW_WBTC_VARIABLE_RATE_SLOPE_2 = 302_00;
+    uint256 public constant NEW_WBTC_RESERVE_FACTOR        = 20_00;
+
+    Domain       mainnet;
+    GnosisDomain gnosis;
+
+    constructor() {
+        id = '20231115';
+    }
+
+    function setUp() public {
+        mainnet = new Domain(getChain('mainnet'));
+        gnosis = new GnosisDomain(getChain('gnosis_chain'), mainnet);
+
+        gnosis.selectFork();
+        new SparkGnosis_20231115();
+
+        mainnet.selectFork();
+
+        payload = deployPayload();
```

Comment:
I think instead of testing with the deployed version on the fork, I would just wait until we have an actual deployed contract on GC or other L2s and put the actual address here. You can determine the correctness of the GC payload by the Gnosis specific test suite.